src/Symfony/Component/Asset/Preload/PreloadManagerInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\Preload;
+
+/**
+ * Manages resources to preload according to the W3C "Preload" specification.
+ *
+ * @see https://www.w3.org/TR/preload/
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface PreloadManagerInterface
```

Comment:
it could also allow something to add a profiler panel reporting the preloaded resources for instance, by using a decorator implementation